client/components/fixed-navigation-header/index.tsx

```diff
@@ -50,20 +50,26 @@ const ActionsContainer = styled.div`
 	align-items: center;
 `;
 
+type NavigationItem = { label: string; href?: string };
 interface Props {
 	id?: string;
 	className?: string;
 	children?: ReactNode;
-	navigationItems: { label: string; href?: string }[];
+	navigationItems: NavigationItem[];
+	mobileItem?: NavigationItem;
 	compactBreadcrumb?: boolean;
 }
 
 const FixedNavigationHeader = React.forwardRef< HTMLElement, Props >( ( props, ref ) => {
-	const { id, className, children, navigationItems, compactBreadcrumb = false } = props;
+	const { id, className, children, navigationItems, mobileItem, compactBreadcrumb = false } = props;
```

Comment:
nitpick: what's your take on removing `navigationItems, mobileItem, compactBreadcrumb = false` in favour of `...breadcrumb` and using it for the props like `breadcrumb.navigationItems` , `breadcrumb.mobileItem`, `breadcrumb.compact`? 

Also, `compactBreadcrumb` is already defaulting to `false` in the `Breadcrumb` component which actually uses it so I suppose we can omit the default value here.

Reply:
Great idea, I am going to try, the `interface Props` of the `FixedNavigationHeader` can be extended to include those `BreadcrumbProps` and using object destructuring we would avoid passing those `props` explicitly here

Reply:
After doing the change, it seems to me more verbose so I left the original version but with some amendments: 
- I have deleted the default value of `compactBreadcrumb` as per your suggestion (good catch)
- I have used the type of the breadcrumb item `TBreadcrumbItem` to avoid type duplication.

Reply:
Great @epeicher, thanks for looking into this. Let's 🚢 it!